README.en.md

```diff
@@ -0,0 +1,37 @@
+# Genius Invokation TCG Simulator
+
+[中文版](./README.md) [日本語版](./README.jp.md)

```

Comment:
All language version should contain all language links like:

```md
[中文版](./README.md) | [日本語版](./README.jp.md) | [English Version](./README.en.md)
```

so that the UX of different langs can be consistent

Reply:
Can ref to https://github.com/Textualize/rich?tab=readme-ov-file